ci/install.sh

```diff
@@ -14,9 +14,9 @@ fi
 
 # Use default CC to avoid compilation problems when installing Python modules.
 echo "Install neovim module for Python 3."
-CC=cc python3 -m pip -q install --user --upgrade pynvim
+CC=cc python3 -m pip -q install --upgrade pynvim
 echo "Install neovim module for Python 2."
-CC=cc python2 -m pip -q install --user --upgrade pynvim
+CC=cc python2 -m pip -q install --upgrade pynvim
```

Comment:
Interesting.
I've tried this in https://github.com/neovim/neovim/pull/9245 - and expected that e.g. `PATH="$HOME/.local/bin:$PATH"` would make it work there.

Reply:
Ah, found https://github.com/neovim/neovim/issues/10270#issuecomment-531516948.
Thanks!